partiql-eval/src/eval.rs

```diff
@@ -614,6 +720,9 @@ pub enum EvalBinOp {
     Div,
     Mod,
     Exp,
+
+    // Boolean ops
+    In,
```

Comment:
fyi, the `feat-value-iter` doesn't have the `IN` operator implementation (https://github.com/partiql/partiql-lang-rust/pull/220), so that PR's diff is included in this PR